internal/wrappers/feature-flags.go

```diff
@@ -26,6 +26,15 @@ var FeatureFlagsBaseMap = []CommandFlags{
 	{
 		CommandName: "cx project create",
 	},
+	{
+		CommandName: "cx import",
+		FeatureFlags: []FlagBase{
```

Comment:
you don't need to set up the FeatureFlags list.
if you use a new FF in your code you have to add it here with default value.
if you don't use the FF at all - you don't need all this item of "cx import"

Reply:
This item must be there because it's used by the uploadsWrapper, used by the import command..